### ISSUE TITLE
[Audio Player]: QA: No visual indication about the selection in 'Playback speed rate'/'Closed Caption'

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9284]
### The problem

Page Element: After selecting any option in 'Playback speed rate'/'Closed Caption'

Issue Description:
After selecting any option, no visual indication about the selection in 'Playback speed rate'/'Closed Caption'

Component(s) impacted
Audio Player

Browser
Firefox

Carbon for IBM.com version
v1.37.0-rc.0

Application/website
Carbon for IBM.com

Package
@carbon/ibmdotcom-web-components

CodeSandbox example
https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### The solution

Expected: Visual indication should be provided.

### Application/website

Carbon for IBM.com

### Business priority

Low Priority = release date is not dependent on fix or not upcoming

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
Design update
- use text to indicate playback speed rate
- add checkmark to menu to indicate which item is selected

![Screen Shot 2022-08-29 at 11 12 56 AM](https://user-images.githubusercontent.com/45692622/187234348-9bc5b6a8-e67d-41a1-a58a-d5d2ec3f608d.png)
